README.md

```diff
@@ -1,52 +1,64 @@
 
-![Running mac osx in a docker container](/running-mac-inside-docker-qemu.png?raw=true "OSX KVM DOCKER")
+![Running Mac OS X in a Docker container](/running-mac-inside-docker-qemu.png?raw=true "OSX KVM DOCKER")
 
-Run Mac in a Docker container! Run near native OSX-KVM in Docker! X11 Forwarding! iMessage security research!
+Run OSX-KVM in Docker with near-native performance! X11 Forwarding! iMessage security research!
 
-Author: Sick.Codes https://sick.codes/ & https://twitter.com/sickcodes
+## Author
 
-Documentation: everything is on this page!
+This project is maintained by [Sick.Codes](https://sick.codes/) [(Twitter)](https://twitter.com/sickcodes)
 
-### PR & Contributor Credits
+Additional credits can be found here: https://github.com/sickcodes/Docker-OSX/blob/master/CREDITS.md
 
-https://github.com/sickcodes/Docker-OSX/blob/master/CREDITS.md
+Additionally, comprehensive list of all contributors can be found here: https://github.com/sickcodes/Docker-OSX/graphs/contributors
 
-Docker Hub: https://hub.docker.com/r/sickcodes/docker-osx
+Special thanks to [@kholia](https://twitter.com/kholia) for maintaining the upstream project, which Docker-OSX is built on top of: [OSX-KVM](https://github.com/kholia/OSX-KVM)
 
-- sickcodes/docker-osx:latest - base recovery image (10)
+## Docker
 
-- sickcodes/docker-osx:big-sur - base recovery image (11)
+Images built on top of the contents of this repository are also available on **Docker Hub** for convenience: https://hub.docker.com/r/sickcodes/docker-osx
 
-- sickcodes/docker-osx:naked - supply your own .img file
+A comprehensive list of the available Docker images and their intended purpose can be found in the [Instructions](README.md#Instructions)
 
-- sickcodes/docker-osx:auto - 17.5GB image boot to OSX shell
+## Kubernetes
 
-## Professional Support Available!
+Docker-OSX supports Kubernetes.
 
-Small questions & issues: open an issue!
+Kubernetes Helm Chart & Documentation can be found under the [helm directory](helm/README.md)
 
-For big projects, DM on Twitter [@sickcodes on Twitter](https://twitter.com/sickcodes) or write to us at https://sick.codes/contact.
+Thanks [cephasara](https://github.com/cephasara) for contributing this major contribution.
 
-- Enterprise support, Business support, or casual support.
-- Custom images, custom scripts, consulting (per hour available!)
-- One-on-one with you, or your development team.
+[![Artifact HUB](https://img.shields.io/endpoint?url=https://artifacthub.io/badge/repository/docker-osx)](https://artifacthub.io/packages/search?repo=docker-osx)
 
-## Kubernetes Support
+## Support
 
-Kubernetes Helm Chart & Documentation [available at ./helm](https://github.com/sickcodes/Docker-OSX/tree/master/helm)
+### Small questions & issues
 
-Thank you to @cephasara for this major contribution.
+Feel free to open an [issue](https://github.com/sickcodes/Docker-OSX/issues/new/choose), should you come across minor issues with running Docker-OSX or have any questions.
```

Comment:
```
Resolved Questions

Many questions have already been asked and answered. Most of the fully resolved answers have made their way back into this page, but feel free to search through the Open and Closed issue list: [https://github.com/sickcodes/Docker-OSX/issues?q=is%3Aissue](https://github.com/sickcodes/Docker-OSX/issues?q=is%3Aissue)
```

Reply:
GitHub already implements a function that shows similar issues that have already been opened or closed based on user input. Is this surely not a bit extra?